projects/driver_controls/src/main.c

```diff
@@ -62,10 +84,11 @@ int main() {
   horn_fsm_init(&fsm_group.horn);
   push_to_talk_fsm_init(&fsm_group.push_to_talk);
 
-  can_fsm_init(&fsm_group.can);
+  GPIOState state;
```

Comment:
This doesn't seem to be used. Should it be?

Reply:
I decided to remove `can_fsm` in favor of having an output function that gets called each time a transition happens.